python/pyarrow/compat.py

```diff
@@ -208,7 +208,10 @@ def import_tensorflow_extension():
             ext = os.path.join(path, "libtensorflow_framework.so")
             if os.path.exists(ext):
                 import ctypes
-                ctypes.CDLL(ext)
+                try:
+                    ctypes.CDLL(ext)
```

Comment:
in the case where this fails, why not try `import tensorflow`?

Reply:
It happens inside of the manylinux1 build (the OSError occurs if the the version of GLIBC is too old for TensorFlow); in this case import tensorflow also fails.

Reply:
Ok, I see.